console/TailwindPlugin.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace Winter\TailwindUI\Console;
+
+use File;
+use Illuminate\Console\Command;
+use System\Classes\PluginManager;
+use Illuminate\Support\Facades\Artisan;
+
+class TailwindPlugin extends Command
+{
+    /**
+     * @var string|null The default command name for lazy loading.
+     */
+    protected static $defaultName = 'tailwind:plugin';
+
+    /**
+     * @var string The name and signature of this command.
+     */
+    protected $signature = 'tailwind:plugin
+        {plugin : Defines plugin to compile tailwind assets}
+        {webpackArgs?* : Arguments to pass through to the Webpack CLI}';
+
+    /**
+     * @var string The console command description.
+     */
+    protected $description = 'Compile plugin tailwind css';
+
+    protected ?string $config = null;
+    protected ?string $original = null;
+    protected ?string $backendCss = null;
+
+    public function __construct()
+    {
+        parent::__construct();
+
+        $dir = plugins_path('winter/tailwindui');
+
+        $this->config = $dir . '/tailwind.config.js';
+        $this->original = $dir . '/tailwind.original.config.js';
+        $this->backendCss = $dir . '/assets/css/dist/backend.css';
+    }
+
+    public function __destruct()
+    {
+        if (File::exists($this->config) && File::exists($this->original)) {
+            File::delete($this->config);
+            File::move($this->original, $this->config);
+        }
+
+        if (File::exists($this->backendCss) && File::exists($this->backendCss . '.original')) {
+            File::delete($this->backendCss);
+            File::move($this->backendCss . '.original', $this->backendCss);
+        }
+    }
+
+    /**
+     * Execute the console command.
+     * @return int
+     */
+    public function handle(): int
+    {
+        $plugin = PluginManager::instance()->findByIdentifier($this->argument('plugin'));
```

Comment:
You can use the `\System\Console\Traits\HasPluginArgument` to add automatic support for autocompletion, validation, and resolution of the plugin argument.